released/packages/coq-bellantonicook/coq-bellantonicook.1.0.0/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
+name: "coq-bellantonicook"
+version: "1.0.0"
+maintainer: "David Nowak <david.nowak@univ-lille.fr>"
+authors: [
+  "Sylvain Heraud"
+  "David Nowak"
+]
+homepage: "https://github.com/davidnowak/bellantonicook"
+dev-repo: "https://github.com/davidnowak/bellantonicook"
+license: "CeCILL-A"
+build: [make]
+build-doc: [make "html"]
+install: [make "install"]
+remove: [make "uninstall"]
+depends: [
+  "coq" {>= "8.6.0"}
```

Comment:
please put an upper bound, eg "8.9.0~", as suggested in the packaging doc https://coq.inria.fr/opam/www/packaging.html